### PR TITLE
haskell-stan: Submission

### PR DESCRIPTION
**Pull request by essandess**
#### Description

This must be merged against #22598.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS x.y
Xcode x.y / Command Line Tools x.y.z

###### Verification 
Have you

- [ ] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [ ] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [ ] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: devel/haskell-stan/Portfile [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                haskell-stan
+version             0.1.0.1
+revision            0
+categories          devel haskell
+maintainers         nomaintainer
+license             Mozilla Public License 2.0
```
Comment (reneeotten) [inline devel/haskell-stan/Portfile:10]: This should be something that MacPorts recognizes. I think it's probable "MPL-2" bit please do a grep in the Port tree to figure it out.

File: devel/haskell-stan/Portfile [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+description         Haskell STatic ANalyser
+
+long_description    Stan is a Haskell STatic ANalysis tool.
+
+homepage            https://github.com/kowainik/stan
+
+set plain_name      stan
```
Comment (reneeotten) [inline devel/haskell-stan/Portfile:18]: no ness for a variable here since you use it only once; just set it directly on the line below.

**Review by reneeotten — CHANGES_REQUESTED** [review]

File: devel/haskell-stan/Portfile (hunk 1 above)
Comment (essandess) [reply devel/haskell-stan/Portfile]: Thanks, you’re correct. Fixed.

File: devel/haskell-stan/Portfile (hunk 2 above)
Comment (essandess) [reply devel/haskell-stan/Portfile]: Thanks, fixed.

**Pull request merged**